### PR TITLE
add image update step to cache warm build

### PR DESCRIPTION
**Pull request by zjaco13**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: platform/backstage/templates/cicd-pipeline/provisioner/wf-templates.yaml [hunk 1]
```diff
@@ -266,5 +281,51 @@ spec:
                 --skip-tls-verify"
                 ]
 
+  - name: update-image-tag
```
Comment (shapirov103) [inline platform/backstage/templates/cicd-pipeline/provisioner/wf-templates.yaml:284]: copy pasted?

**Review by shapirov103 — COMMENTED** [review]
Thanks for the quick turn around. curious if this workflow allows any reuse mechanism. I assume it is a copy paste, but curious if the workflows become more complex whether we can reuse steps between workshops.

File: platform/backstage/templates/cicd-pipeline/provisioner/wf-templates.yaml (hunk 1 above)
Comment (zjaco13) [reply platform/backstage/templates/cicd-pipeline/provisioner/wf-templates.yaml:284]: yes it was.  I already have this cache warmer as a template, but now that its the same as the cicd workflow I can probably reuse there

Comment (zjaco13) [reply platform/backstage/templates/cicd-pipeline/provisioner/wf-templates.yaml:284]: will look into

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**